selfdrive/ui/qt/api.h

```diff
@@ -31,7 +31,7 @@ class HttpRequest : public QObject {
   bool timeout() const;
 
 signals:
-  void requestDone(const QString &response, bool success);
+  void requestDone(const QString &response, bool success, QNetworkReply::NetworkError error);
```

Comment:
extra arguments are safely ignored, so no changes to the other code is needed for this.